connect-inject/endpoints_controller_test.go

```diff
@@ -2230,14 +2494,19 @@ func createPod(name, ip string, inject bool) *corev1.Pod {
 		Status: corev1.PodStatus{
 			PodIP:  ip,
 			HostIP: "127.0.0.1",
+			Phase:  corev1.PodRunning,
+			Conditions: []corev1.PodCondition{{
+				Type:    corev1.PodReady,
+				Status:  corev1.ConditionFalse,
+				Message: testFailureMessage,
+			}},
 		},
 	}
 	if inject {
-		pod.Labels[labelInject] = injected
+		pod.Labels[annotationStatus] = injected
```

Comment:
I realize now after seeing this that `annotationStatus` could probably have a better name since it's used for both annotation and label now. Not a blocker though but something to address at some point.

Reply:
As I was going through all of the places to change this it occurred to me its pretty wonky.
The label const doesn't belong in annotations.go either :(

I'll try to clean it up before GA if we can come to an agreement on where it belongs?